ToDoList/src/ToDoList.API.DependencyInjection/Properties/AssemblyInfo.cs

```diff
@@ -0,0 +1,36 @@
+﻿using System.Reflection;
+using System.Runtime.CompilerServices;
+using System.Runtime.InteropServices;
+
+// General Information about an assembly is controlled through the following
+// set of attributes. Change these attribute values to modify the information
+// associated with an assembly.
+[assembly: AssemblyTitle("ToDoList.API.DependencyInjection")]
```

Comment:
This was first assembly info with `API` in title, so the comment goes here 👼 
Can you please rename all `API` projects to `Api` – it is essential in (default) namespaces, but changing it everywhere does not seem to be that extra work and looks more consistent…